### PR TITLE
Unblock tsdb tracks for public serverless

### PR DESCRIPTION
**Pull request by gbanasiak**
Similar to https://github.com/elastic/rally-tracks/pull/465.

Remarks:
* `downsample` challenge in `tsdb` track doesn't work because downsample operations get skipped by Rally (downsampling not available for public use yet, see [here](https://github.com/elastic/elasticsearch/blob/ecd13e3f118d34e151081cd1e3d432a609295dad/x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/RestDownsampleAction.java#L25)),
* in `tsdb_k8s_queries` track index settings fall into 2 categories: the ones not available in serverless at all (e.g. `lifecycle.name`), and the ones that are available through operator only (e.g. `number_of_shards`).

### CODE REVIEW

File: tsdb/README.md
Comment (martijnvg) [inline tsdb/README.md:190]: What is the reason behind this post ingest sleep?

**Review by martijnvg — APPROVED** [review]
One question. LGTM.

File: tsdb/README.md
Comment (gbanasiak) [reply tsdb/README.md:190]: Pre-serverless benchmarks often force merge after write (sometimes down to a single segment) and then wait until the merge operation finishes by verifying indexing stats periodically. Example [here](https://github.com/elastic/rally-tracks/blob/9293eea47bc850c4041424d59c83352c03692b2e/tsdb/challenges/default.json#L75-L95). The intention of this sequence was increased repeatability of tests.

Neither the force merge, nor index stats are available in serverless (Rally auto-skips those when it detects serverless endpoint) so we're providing a naive but optional alternative in the form of a configurable sleep. We don't have empirical evidence it's helping (sleep long enough should) but since we're going through majority of tracks it looked reasonable to add something. As we are also adding unique markers, this can be later mass-edited, if need be.

Comment (martijnvg) [reply tsdb/README.md:190]: Thanks for the explanation. Makes sense.

**Pull request merged**
